Extend thumbnails section with version specifications and caution (#557)
saleor/saleor-docs: docs/developer/thumbnails.mdx

the `size` becomes required as well. If you don't specify these or pass only one,
the original image attributes will be used.

:::caution

> You might meet the problem with the `WebP` content type recognition.

Reviewer: ```suggestion
You may have encountered a problem with the `WebP` content type recognition.
```
